tests/robottelo/test_decorators.py

```diff
@@ -222,7 +222,7 @@ def bomb(_):
 
 
 class GetBugzillaBugStatusIdTestCase(TestCase):
-    """Tests for :func:`robottelo.decorators._get_bugzilla_bug`."""
+    """Tests for ``robottelo.decorators._get_bugzilla_bug``."""
```

Comment:
```py:func reference target not found: robottelo.decorators._get_bugzilla_bug```
As we are not documenting private functions, they can't be referenced with `:func:`